capstone-frontend/src/components/BookSearchList.jsx

```diff
@@ -19,15 +19,15 @@ class BookSearchList extends Component {
       .catch(err => console.log(err));
   }
 
-  getUserBookLists = () => {
+  getUserBookLists = async () => {
 
-    const userID = window.sessionStorage.getItem("userID");
+    const userID = window.localStorage.getItem("userID");
 
-    fetch(`/api/booklist?userID=${userID}`, {
+    const bookLists = await fetch(`/api/booklist?userID=${userID}`, {
       method: "GET",
-    }).then(response => response.json()).then(res => {
-      this.setState({ bookLists: res })
-    });
+    }).then(resp => resp.json());
+
+    this.setState({ bookLists });
```

Comment:
What was the motivation for changing this to use await versus promise chaining?

Reply:
I'd like to stay consistent across files/components in how we do our fetches, we can discuss together what we think is best.

Reply:
Personally, I think async and await make the code more readable. I don't think its necessary to keep fetches consistent.